clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -382,5 +385,31 @@ private AbstractRequestResponse createUpdateMetadataResponse() {
         return new UpdateMetadataResponse(Errors.NONE.code());
     }
 
+    private AbstractRequest createCreateTopicRequest() {
+        CreateTopicRequest.TopicDetails request1 = new CreateTopicRequest.TopicDetails(
+            3, 5, new HashMap<Integer, List<Integer>>(), new HashMap<String, String>());
 
+        Map<Integer, List<Integer>> replicaAssignments = new HashMap<Integer, List<Integer>>();
```

Comment:
I had considered this but assumed there was a reason even newly added Requests/Responses were using the more explicit pattern and figured it did not hurt. I am happy to remove it. Should we then remove all instances of this at some point?